db/seeds/dummy_structures.rb

```diff
@@ -1,32 +1,8 @@
 # frozen_string_literal: true
 
-SchoolDataImporter.new(Rails.logger).delay.run
-
-if Cohort.none?
-  Cohort.find_or_create_by!(start_year: 2021)
-  Cohort.find_or_create_by!(start_year: 2022)
-end
-
-# TODO: Remove this when we have a way of adding lead providers, or expand to include all of them
-unless LeadProvider.first
-  LeadProvider.find_or_create_by!(name: "Test Lead Provider")
-end
-
-test_lead_provider = LeadProvider.find_by(name: "Test Lead Provider")
-
-if test_lead_provider
-  test_lead_provider.cohorts = Cohort.all
-  test_lead_provider.save!
-end
-
-if CoreInductionProgramme.none?
-  CoreInductionProgramme.find_or_create_by!(name: "Ambition Institute")
-  CoreInductionProgramme.find_or_create_by!(name: "Education Development Trust")
-  CoreInductionProgramme.find_or_create_by!(name: "Teach First")
-  CoreInductionProgramme.find_or_create_by!(name: "UCL")
-end
-
 if Rails.env.development? || Rails.env.deployed_development?
+  SchoolDataImporter.new(Rails.logger).delay.run
+
```

Comment:
Soo, since this won't run on seed, will we have to run it manually first? Or will we wait till next day to check if it worked?

Reply:
We don't need to know about schools in production yet, and I'm not confident that it won't change before we do. I'm hoping to simplify any potential migrations by not having any school data. We can move this to the other seed file when we're ready

Reply:
Right, but we have a cron job that will import them every night, right? Am I missing something? Will we disable it?

Reply:
You never triggered that to actually run...

Reply:
As far as I can see, we never call [schedule](https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/models/cron_job_scheduler.rb#L8)

Reply:
Ooops, you are right 😂 I didn't follow the trail to the end...

I guess that means we don't need to disable it.

Reply:
We just need to not enable it. Which is much easier

Reply:
I mean, we need to remember not to enable it by accident when we add some other cron jobs, if we ever do